test/test-tty-duplicate-key.c

```diff
@@ -0,0 +1,313 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
For new files, this should be:
```C
/* Copyright libuv project contributors. All rights reserved.
```

unless this test file is heavily based on an existing test file.